gem2rpm/smart_proxy_plugin.spec.erb

```diff
@@ -11,15 +11,15 @@ has_cronjob = spec.files.any? { |f| f =~ %r{\Aextra/.*\.cron\z } }
 
 %{!?_root_bindir:%global _root_bindir %{_bindir}}
 %{!?_root_datadir:%global _root_datadir %{_datadir}}
-%{!?_root_localstatedir:%global _root_localstatedir %{_localstatedir}}
+%{!?_root_sharedstatedir:%global _root_sharedstatedir %{_sharedstatedir}}
 %{!?_root_sysconfdir:%global _root_sysconfdir %{_sysconfdir}}
 
 %global gem_name <%= spec.name %>
 %global plugin_name <%= spec.name.sub(/\Asmart_proxy_/, '') %>
 
 %global foreman_proxy_min_version 1.24
 %global foreman_proxy_dir %{_root_datadir}/foreman-proxy
-%global foreman_proxy_statedir %{_root_localstatedir}/foreman-proxy
+%global foreman_proxy_statedir %{_root_sharedstatedir}/foreman-proxy
 %global foreman_proxy_bundlerd_dir %{foreman_proxy_dir}/bundler.d
 %global foreman_proxy_settingsd_dir %{_root_sysconfdir}/foreman-proxy/settings.d
 %global smart_proxy_dynflow_bundlerd_dir %{!?scl:/opt/theforeman/tfm/root}%{_datadir}/smart_proxy_dynflow_core/bundler.d
```

Comment:
Unrelated, but @zjhuntin this looks like it might break on EL8

Reply:
Why is that?

Reply:
Doesn't this mean it'll be prefixed with `/opt/theforeman/tfm/root` on EL8?

Reply:
Seems to be working as expected if you compare:

http://koji.katello.org/koji/rpminfo?rpmID=340390
http://koji.katello.org/koji/rpminfo?rpmID=340393